claat/proto-renderer/testing-utils/proto_constructors.go

```diff
@@ -0,0 +1,35 @@
+package testingUtils
+
+import (
+	"github.com/googlecodelabs/tools/third_party"
+)
+
+// Helper constructor functions
+
+func NewStylizedTextPlain(txt string) *tutorial.StylizedText {
+	return &tutorial.StylizedText{
+		Text: txt,
+	}
+}
+
+func NewStylizedTextStrong(txt string) *tutorial.StylizedText {
+	return &tutorial.StylizedText{
+		Text:     txt,
+		IsStrong: true,
+	}
+}
+
+func NewStylizedTextEmphazied(txt string) *tutorial.StylizedText {
```

Comment:
typo

Reply:
`Emphazied` => `Emphasized`? that's it?
could you explicitly point out the typo next time? 🙈